packages/components/locales/si-LK.json

```diff
@@ -51,7 +51,7 @@
   },
   "kzErrorPage.404.message" : {
     "description" : "Call to action instructions for the user",
-    "message" : "කණගාටුයි, නමුත් අපි ඔබ සොයන පිටුව fing නොහැක.  ආපසු ගොස් නැවත උත්සාහ කරන්න, නැතහොත් නිවසට යන්න"
+    "message" : "කණගාටුයි, නමුත් අපි ඔබ සොයන පිටුව find නොහැක.  ආපසු ගොස් නැවත උත්සාහ කරන්න, නැතහොත් නිවසට යන්න"
```

Comment:
Omg lmao 🫠 Okay maybe this one needs to actually be re-translated.

Reply:
haha yeah okay, it's been removed for re-translation.